blogs/ecosystems-and-partners/zyphra/README.md

```diff
@@ -0,0 +1,125 @@
+---
+blogpost: true
+date: 10 December 2024
+blog_title: "Zyphra Introduces Frontier Training Kernels for Transformers and SSMs on AMD Instinct MI300X Accelerators"
+thumbnail: 'amd_zyphra.jpg'
+author: Quentin Anthony
+tags: LLM, AI/ML, GenAI, PyTorch
+category: Ecosystems and Partners
+language: English
+myst:
+  html_meta:
+    "description lang=en": "In this blog, we demonstrate the first backwards kernels to surpass H100s for both transformers (Flash Attention v2) and hybrid models (Mamba2), which enables training foundation models on AMD Instinct MI300X accelerators."
+    "keywords": "Mamba, PyTorch, S4, S6, Mamba2, Transformer, Flash Attention, Optimization, Hardware-aware, Transformer, Attention, ROCm, Mi210, MI250, MI300, AI/ML, Generative AI"
+    "property=og:locale": "en_US"
+---
+
+
+# Training Transformers and Hybrid models on AMD Instinct MI300X Accelerators
```

Comment:
Reverting back to an earlier suggested title, please use:
```suggestion
# Zyphra Speeding Up Training on AMD Instinct MI300X Accelerators
```

Reply:
We prefer the previous title for this one as well